### PR TITLE
[ACS-12607] added error card and legend to the teams card

### PR DESCRIPTION
**Pull request by adamswiderskipw**
**JIRA ticket link or changeset's description**
https://hyland.atlassian.net/browse/ACS-12607

### CODE REVIEW

File: .github/scripts/e2e-teams-report.js [hunk 1]
```diff
@@ -338,6 +346,77 @@ function buildCard({ suiteStats, browserStats, browserSuiteStats }) {
...
+    { browser: 'msedge', result: process.env.MSEDGE_RESULT }
+  ].filter((job) => job.result);
+
+  const mention = buildMention();
+
+  const body = [
+    { type: 'TextBlock', size: 'Large', weight: 'Bolder', text: `ACA Cron Multibrowser Workflow - ${date}` },
+    {
+      type: 'TextBlock',
+      text: `🛑 Setup failed — no tests were run${mention ? ` — ${mention.tag}` : ''}`,
```
[the remark this answers is not in the document]
Comment (adamswiderskipw) [reply .github/scripts/e2e-teams-report.js:385]: no important and an edge case

**Review by akashrathod28 — APPROVED** [review]
